### PR TITLE
Added likes and dislikes column to eco news management panel (#4686).

### LINKED ISSUE #4686: change rating field into likes/dislikes on eco news page

### PR DESCRIPTION
**Pull request by lena00-afk**
- add dislike endpoint
- list eco news likes/dislikes
- view likes/dislikes from eco news panel
- code fix

### CODE REVIEW

File: core/src/test/java/greencity/controller/EcoNewsControllerTest.java [hunk 1]
```diff
@@ -245,6 +243,20 @@ void likeTest() throws Exception {
         verify(ecoNewsService).like(userVO, 1L);
     }
 
+    @Test
+    void dislikeTest() throws Exception {
+        // given
+        UserVO userVO = getUserVO();
+        // when
+        when(userService.findByEmail(anyString())).thenReturn(userVO);
```
Comment (PawelMiskowiec) [inline core/src/test/java/greencity/controller/EcoNewsControllerTest.java:250]: I think this belongs to given

**Review by PawelMiskowiec — APPROVED** [review]

File: core/src/main/resources/templates/core/management_eco_news.html [hunk 2]
```diff
@@ -40,6 +40,25 @@
...
+            display: inline-block;
+            cursor: pointer;
+            color: #ffffff !important;
+            font-size: 13px;
+            padding: 0px 8px;
+            text-decoration: none;
+        }
+        .open-likesButton:hover {
+            background-color: #1dcf6d;
+        }*/
```
Comment (F1iper) [inline core/src/main/resources/templates/core/management_eco_news.html]: Seems like an unnecessary comment

**Review by F1iper — CHANGES_REQUESTED** [review]

**Review by macuzarski — APPROVED** [review]

File: core/src/test/java/greencity/controller/EcoNewsControllerTest.java (hunk 1 above)
Comment (lena00-afk) [reply core/src/test/java/greencity/controller/EcoNewsControllerTest.java:250]: fixed in new commit

File: core/src/main/resources/templates/core/management_eco_news.html (hunk 2 above)
Comment (lena00-afk) [reply core/src/main/resources/templates/core/management_eco_news.html]: fixed :)

**Review by F1iper — APPROVED** [review]

**Pull request merged**